src/components/pages/drs/DrsIndex.js

```diff
@@ -0,0 +1,29 @@
+import '@fontsource/roboto';
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+  Switch,
+  Route,
+  useRouteMatch
+} from "react-router-dom";
+import DrsShow from './DrsShow'
+
+const DrsIndex = () => {
+  let match = useRouteMatch();
+  return(
+    <div align="center">
+      <meta
+          name="viewport"
+          content="minimum-scale=1, initial-scale=1, width=device-width"
+      />
+      <Typography variant="h2">Welcome to DRS Starter Kit</Typography>
+      <Switch>
```

Comment:
what is this `Switch` doing? the `DrsShow` route is already captured by the top layer router, it doesn't need to be here as well